DoxygenHelp.md

```diff
@@ -22,27 +22,37 @@ If DOXYGEN_GENERATE_DOCUMENTATION is selected in the CMake GUI, CMake will autom
 https://github.com/tumcms/Open-Infra-Platform/blob/development/CMakeLists.txt
 
 In the CMake GUI, the following options can be selected:
-- DOXYGEN_OPTIONAL_COMMENTED_ONLY: If this is not selected, doxygen will assume that all entities are documented, even if there is no documentation available. 
-- DOXYGEN_OPTIONAL_INCLUDE_INTERNAL: If selected, this will include all comments preceded by "\internal" (comments intended for developers only).
-- DOXYGEN_OPTIONAL_INCLUDE_EARLYBINDING: If selected (... and already built), this will include the selected IFC schemas in the documentation. 
+- DOXYGEN_OPTIONAL_INCLUDE_COMMENTED_ONLY: If selected, only documentation for explicitly commented entities will be generated. Otherwise, doxygen will assume that all entities are documented, even if there is no documentation available. 
+- DOXYGEN_OPTIONAL_INCLUDE_INTERNAL: If selected, this will include all comments preceded by "\internal". These are comments intended for developers only. So if the generated documentation is intended for external users, this option should not be selected.
+- DOXYGEN_OPTIONAL_INCLUDE_EARLYBINDING: If selected (... and already built), this will include the selected IFC schemas in the documentation. Otherwise, the schemas will not be included in the documentation.
+- DOXYGEN_OPTIONAL_AUTO_OPEN_DOCUMENTATION: If selected, the index.html page of the documentation will automatically open. If not selected, you will have to find and open the generated documentation manually (see section below on building the documentation). 
 
 ### ConfigureDoxygen.cmake file
 If DOXYGEN_GENERATE_DOCUMENTATION is selected in CMake, CMake will include ConfigureDoxygen.cmake. This file includes all settings for the generation of doxygen documentation for the Open Infra Platform project. Changes to documentation settings should be made here. doxygen_add_docs() creates the GenerateDocumentation target. The up-to-date version of the file can be found here:
 
 https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake
 
-A tag that is more likely to be required to change:
-- DOXYGEN_EXCLUDE_PATTERNS: if you want to exclude certain source code directories or files from the documentation, you should add them here.
+#### ToDos.
+DOXYGEN_EXCLUDE_PATTERNS is a tag that is more likely to be required to change.
+It excludes certain source code directories or files from the documentation. In this project, the following directories including the following patterns are currently excluded:
+- */CurrentlyExcluded/* 
+- */cmake/*
+- */deploy/*
+- */Documentation/* 
+- */external/*
+- */testdata/*
+- */Tools/* 
+- */UserInterface/QtPropertyBrowser/*

```

Comment:
The */.../* is Doxy specific syntax. Perhaps mention it in a sentence, but list the folders normally (e.g. Documentation/*).

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/db191d9d975f8d80b167ef09875962d736ab5077